traveler-ios-ui/TravelerKitUI/Storyboards/Parking.storyboard

```diff
@@ -424,29 +426,29 @@
-                                                            <rect key="frame" x="15" y="115.5" width="40" height="19.5"/>
+                                                            <rect key="frame" x="14.999999999999996" y="115.66666666666667" width="39.666666666666657" height="19.333333333333329"/>
                                                             <fontDescription key="fontDescription" type="system" pointSize="16"/>
                                                             <nil key="textColor"/>
                                                             <nil key="highlightedColor"/>
                                                         </label>
                                                         <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Label" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="NhD-ER-ni2">
-                                                            <rect key="frame" x="63" y="115.5" width="41.5" height="19.5"/>
+                                                            <rect key="frame" x="62.666666666666671" y="115.66666666666667" width="41.333333333333329" height="19.333333333333329"/>
                                                             <fontDescription key="fontDescription" type="system" weight="semibold" pointSize="16"/>
                                                             <nil key="textColor"/>
                                                             <nil key="highlightedColor"/>
                                                         </label>
                                                         <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="roundedRect" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="u5v-8y-tV9">
-                                                            <rect key="frame" x="336" y="110.5" width="44" height="30"/>
+                                                            <rect key="frame" x="336" y="110.33333333333333" width="44" height="29.999999999999986"/>
                                                             <state key="normal" title="View ›"/>
                                                             <connections>
                                                                 <action selector="didView:" destination="UYw-M5-Sw6" eventType="touchUpInside" id="wNu-6a-3eI"/>
-                                                                <segue destination="8o5-BF-1H0" kind="show" identifier="detailsSegue" id="BEs-HM-xAb"/>
+                                                                <segue destination="8o5-BF-1H0" kind="show" identifier="detailsSegue" id="URB-uu-SN5"/>
```

Comment:
You have two segues with the same identifier in the storyboard. Both coming out of the view controller. This is unnecessary. If both your button and your cell are going to do something then just have the cell do it. If both take you to the same place, it going to be unnoticeable to the user the difference between tapping the cell and tapping the button.